src/main/kotlin/app/utils/EmailValidator.kt

```diff
@@ -0,0 +1,21 @@
+package app.utils
+
+import com.beust.jcommander.IParameterValidator
+import com.beust.jcommander.ParameterException
+
+object EmailValidator : IParameterValidator {
+    @Throws(ParameterException::class)
+    override fun validate(name: String, value: String) {
+        if (!isValidEmail(value)) {
+            throw ParameterException("Parameter $name should be "
+                    + "correct email (found $value)")
+        }
+    }
+
+    fun isValidEmail(email: String): Boolean {
+        val ePattern = "^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@((\\[[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}\\])|(([a-zA-Z\\-0-9]+\\.)+[a-zA-Z]{2,}))$"
```

Comment:
Is there a way to bread down this line? Also, seriously we're using IP addresses for email hosts? I think we can safely skip that.